hello-server.ts

```diff
@@ -0,0 +1,13 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+
+function handler(req: IncomingMessage, res: ServerResponse) {
+  res.writeHead(200, { "Content-Type": "text/plain" });
+  res.end(`Hello, world!`);
+}
+
+const server = createServer(handler);
```

Comment:
We create an HTTP server using createServer(handler), which tells Node.js to run the handler function for every request.